cfme/tests/cloud_infra_common/test_events.py

```diff
@@ -38,7 +38,7 @@ def vm_crud(provider, setup_provider_modscope, small_template_modscope):
 @pytest.mark.uncollectif(lambda provider: provider.one_of(RHEVMProvider) and
                          BZ(1509020, forced_streams=['5.8', '5.9']).blocks,
                          'BZ 1509020')
```

Comment:
@izapolsk Could you please clarify what is the relation between this bug and the one I used as a blocker. They seem to be dealing with the same thing but are not considered duplicate.

Reply:
as far as I remember I submitted a BZ that rhevm's "create vm" EmsEvent event is ignored and cfme doesn't emit appropriate MiqEvent. I also attached logs. When developers started working on it, they found a bug in logs related to retirement. So, that my bug was clonned and one of BZ records was renamed.